src/main/scala/org/portablescala/sbtplatformdeps/PlatformDepsBuilders.scala

```diff
@@ -0,0 +1,52 @@
+package org.portablescala.sbtplatformdeps
+
+import sbt._
+
+import scala.language.experimental.macros
+import scala.reflect.macros.Context
+
+final class PlatformDepsGroupID private[sbtplatformdeps] (
+    private val groupID: String) {
+  def %%%(artifactID: String): PlatformDepsGroupArtifactID =
```

Comment:
I'm assuming there will also (later) be a more generalized form of `%%%!` or at least something that lets one create `ModuleID`s without requiring to be in an sbt macro context.

Reply:
Basically there is already such a thing: using `"org.foo" % "bar" % "1.0" cross ScalaJSCrossVersion.binary`. It's barely less convenient than `%%%!`, and definitely more readable. Note that in sbt-crossproject we had already dumped `%%%!`. sbt-crossproject and sbt-platform-deps can't know anything about specific platforms, so if a platform wants a more convenient way to build its `ModuleID`s, it needs to provide it on its own. But I wouldn't do that anymore. The `cross` thing seems sufficient.

Reply:
Fair enough.